config/jail.conf

```diff
@@ -767,3 +767,18 @@ port = 2222
 enabled  = false
 logpath  = /var/lib/portsentry/portsentry.history
 maxretry = 1
+
+[pass2allow]
+# allow FTP traffic after successful HTTP auth
+enabled    = false
```

Comment:
default fail2ban does not have any jails enabled, why would you default this jail to be enabled?